### ISSUE TITLE
Show a given package only once among Trending / Recent / Most Used?

### ISSUE THREAD

**Issue by dandv** [percolatestudio/atmosphere#347]
While a given package may very well be trending and recent (-ly updated) and Most Used, would it be better for diversity's sake to feature packages in only one of the categories?

![image](https://cloud.githubusercontent.com/assets/33569/6282169/318d9460-b885-11e4-9cac-c58673cbbeb6.png)

**Comment by zol** [comment]
Agreed. We should probably just show them in the 'Trending' section.

**Comment by tmeasday** [comment]
Hmmm. It's a good idea but it presents some implementation challenges ;)
